CHANGELOG.rst

```diff
@@ -101,6 +101,90 @@ Added
 
 Changed
 ^^^^^^^
+* `@senwu`_: Switch to Emmental as the default learning engine.
+
+.. note::
+
+    Instead of owning Fonduer learning engine, we switch to Emmental which is a deep learning
+    framework for multi-task learning. This change can give Fonduer more power to support more
+    applications. Example usage:
+
+    .. code:: python
+
+        import emmental
+
+        # Collect word counter from candidates
+        word_counter = collect_word_counter(train_cands)
+
+        emmental.init(fonduer.Meta.log_path)
+
+        # Geneate special tokens
+        arity = 2
+        specials = []
+        for i in range():
+            specials += [f"~~[[{i}", f"{i}]]~~"]
+
+        # Generate word embedding module
+        emb_layer = EmbeddingModule(
+            word_counter=word_counter, word_dim=300, specials=specials
+        )
+
+        # Only use the samples that have labels
+        diffs = train_marginals.max(axis=1) - train_marginals.min(axis=1)
+        train_idxs = np.where(diffs > 1e-6)[0]
+
+        # Build training dataloader
```

Comment:
Similarly, would be nice to have a brief mention of the why for these comments. I see that we're declaring a EmmentalDataLoader, but why do I need it? What does it to? This will help a user understand what this is replacing in the old code.

E.g., Create a dataloader to <purpose>